.github/workflows/build-test-actions.yml

```diff
@@ -28,10 +28,8 @@ jobs:
       - name: Run pylint
         run: poetry run pylint --exit-zero toqito/
```

Comment:
Might be better to remove this `pylint` check as `ruff` lets us control what is checked based on the available rules. 

https://docs.astral.sh/ruff/rules/#pyflakes-f

The workflow finally flagged lines>120 as errors due to `ruff`.

Reply:
Yep, perfect, let's kill the `pylint` line.

And while that's probably going to lead to lots of flagged lines, it's great that it's catching! Think we can use `black` to auto-format the lines of offending files for length to automate that process.

Reply:
I think we should change our mind about removing `pylint`. :grimacing:

I just realized `ruff` does not implement all the `pylint` rules yet. https://github.com/astral-sh/ruff/issues/970

For example, the following earnings are raised by pylint but not by ruff. 

![image](https://github.com/vprusso/toqito/assets/66048318/244998f4-1008-4e94-8f61-345b43193faa)

Reply:
Good call. All for that! 👍